routers/init.go

```diff
@@ -167,6 +169,7 @@ func GlobalInitInstalled(ctx context.Context) {
 func NormalRoutes(ctx context.Context) *web.Route {
 	ctx, _ = templates.HTMLRenderer(ctx)
 	r := web.NewRoute()
+	r.Use(chi_middleware.RealIP)
```

Comment:
That's incorrect. Attacker will flood Gitea with a lot of fake IPs by filling the X-Real-IP header.

Reply:
And the RealIP's document said:

```
// You should only use this middleware if you can trust the headers passed to
// you (in particular, the two headers this middleware uses), for example
// because you have placed a reverse proxy like HAProxy or nginx in front of
// chi. If your reverse proxies are configured to pass along arbitrary header
// values from the client, or if you use this middleware without a reverse
// proxy, malicious clients will be able to make you very sad (or, depending on
// how you're using RemoteAddr, vulnerable to an attack of some sort).
```

Reply:
Changed to internal routes only.